benchbuild/utils/run.py

```diff
@@ -5,14 +5,20 @@
 from contextlib import contextmanager
 
 import attr
+import plumbum as pb
 from plumbum import TEE, local
 from plumbum.commands import ProcessExecutionError
 
 from benchbuild import settings, signals
 
+if t.TYPE_CHECKING:
+    import mypy_extensions as mypy
```

Comment:
Codacy found an issue: [Unable to import 'mypy_extensions'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)